cherab/adas/repository/atomic.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016-2018 Euratom
+# Copyright 2016-2018 United Kingdom Atomic Energy Authority
+# Copyright 2016-2018 Centro de Investigaciones Energéticas, Medioambientales y Tecnológicas
+#
+# Licensed under the EUPL, Version 1.1 or – as soon they will be approved by the
+# European Commission - subsequent versions of the EUPL (the "Licence");
+# You may not use this work except in compliance with the Licence.
+# You may obtain a copy of the Licence at:
+#
+# https://joinup.ec.europa.eu/software/page/eupl5
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the Licence is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the Licence for the specific language governing permissions and limitations
+# under the Licence.
+
+
+import cherab.openadas.repository.atomic as atomic
+from cherab.openadas.repository.atomic import *
+from .utility import DEFAULT_REPOSITORY_PATH
+
+
+atomic.DEFAULT_REPOSITORY_PATH = DEFAULT_REPOSITORY_PATH
```

Comment:
I don't think we should monkey-patch modules in `cherab.openadas` with the ADAS repository path. This could cause hard-to-find bugs if users somehow tried or wanted to use OpenADAS and ADAS data in the same program. All the functions in `cherab.openadas.repository` accept a `repository_path` argument: I think a more robust way to deal with this is to simply wrap all the functions in the `cherab.openadas.repository` module with `repository_path=cherab.adas.repository.utility.DEFAULT_REPOSITORY_PATH`. For this, [functools.wraps](https://docs.python.org/3/library/functools.html#functools.wraps) will be useful.

Reply:
Yes, I've used the laziest possible solution. Your solution is better. Now all the functions from `cherab.openadas.repository` are wrapped in `repository_path_decorator`. Please, let me know if this wrapper can be improved.